bin/detail/call.py

```diff
@@ -63,15 +63,16 @@ def call(call_args, logging, cache_file='', ignore=False, sleep=0):
   for i in call_args:
     pretty += '  `{}`\n'.format(i)
   pretty += ']\n'
-  print(pretty)
+  if logging.verbosity == 'full':
+    print(pretty)
   logging.log_file.write(pretty)
 
   # print one line version
   oneline = ''
   for i in call_args:
     oneline += ' "{}"'.format(i)
   oneline = "[{}]>{}\n".format(os.getcwd(), oneline)
-  if logging.verbose:
+  if logging.verbosity == 'full':
```

Comment:
Is this information really required in the `normal` case? For example as a developer you either checked out a project and want to build it, or just doing your routine  rebuilds, are the command invocation details really helpful in this case?